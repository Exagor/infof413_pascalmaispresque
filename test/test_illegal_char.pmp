begin
    read(a)...
    print(A)...
    print(?)...
    print(78)...
end