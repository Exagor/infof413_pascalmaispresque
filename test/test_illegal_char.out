token: begin	lexical unit: BEG
token: read	lexical unit: READ
token: (	lexical unit: LPAREN
token: a	lexical unit: VARNAME
token: )	lexical unit: RPAREN
token: print	lexical unit: PRINT
token: (	lexical unit: LPAREN
Illegal character: A
token: )	lexical unit: RPAREN
token: print	lexical unit: PRINT
token: (	lexical unit: LPAREN
Illegal character: ?
token: )	lexical unit: RPAREN
token: print	lexical unit: PRINT
token: (	lexical unit: LPAREN
token: 78	lexical unit: NUMBER
token: )	lexical unit: RPAREN
token: end	lexical unit: END
Variables :
a   2
